orderer/common/cluster/clusterservice.go

```diff
@@ -59,7 +59,7 @@ type ClusterService struct {
 
 type AuthRequestSignature struct {
 	Version        int64
-	Timestamp      string
+	Timestamp      []byte
```

Comment:
You can just put here int64, and then you don't need EncodeTimestamp at all

Reply:
IMO we will still need it, because:
1. We want to avoid bugs such as `timestamp.Nano `.
2. We have at least 2 places where we need to convert the timestamp to byte array.

Reply:
right now you're encoding the seconds and not the nano seconds: `binary.LittleEndian.PutUint64(b, uint64(t.Seconds))` so what's the difference from taking the wrong time from the struct? 

This struct is only used for signing and verifying a signature, I don't understand where you convert it to a byte array?

Reply:
Just saying that we still need the function (to convert timestamp/uint64 to a byte array).
We use the byte array to calculate the session binding hash on both ends.